staking/apr/compute.go

```diff
@@ -0,0 +1,101 @@
+package apr
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/harmony-one/harmony/block"
+	"github.com/harmony-one/harmony/core/state"
+	"github.com/harmony-one/harmony/shard"
+	staking "github.com/harmony-one/harmony/staking/types"
+	"github.com/pkg/errors"
+)
+
+// Reader ..
+type Reader interface {
+	GetHeaderByNumber(number uint64) *block.Header
+	ReadValidatorSnapshotAtEpoch(
+		epoch *big.Int,
+		addr common.Address,
+	) (*staking.ValidatorWrapper, error)
+}
+
+const (
+	nanoSecondsInYear = time.Nanosecond * 3.154e+16
```

Comment:
why use nanoseconds? The header.Time() is in Unix seconds.

Reply:
I thought I saw it used nano